frontend/src/app/app.module.ts

```diff
@@ -2,13 +2,18 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppComponent } from './app.component';
+import { TimezonesComponent } from './timezones/timezones.component';
+
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
**nitpick:** Is there a standard for how these imports are ordered in TS? I would expect to see this one with the other `@angular/...` import above the local imports.

Reply:
I just added Prettier with a plugin that organizes TypeScript imports. Before I commit and push it, do you have an opinion on how you'd want Prettier to run? I'm thinking pre-commit hook, but I'm open to other ideas.